server/src/main/java/org/opensearch/cluster/routing/remote/InternalRemoteRoutingTableService.java

```diff
@@ -300,4 +301,15 @@ protected void doStart() {
     @Override
     protected void doStop() {}
 
+    @Override
+    public void deleteStaleIndexRoutingPaths(List<String> stalePaths) throws IOException {
+        try {
+            logger.debug(() -> "Deleting stale index routing files from remote - " + stalePaths);
+            blobStoreRepository.blobStore().blobContainer(BlobPath.cleanPath()).deleteBlobsIgnoringIfNotExists(stalePaths);
+        } catch (IOException e) {
+            logger.error(() -> new ParameterizedMessage("Failed to delete some stale index routing paths " + "from {}", stalePaths), e);
```

Comment:
Why do we need `+` here: ` paths " + "from `

Reply:
I did break line in IDE, which resulted in this. Will remove it.